org.eclipse.lyo.oslc4j.codegenerator/src/org/eclipse/lyo/oslc4j/codegenerator/files/generateAdaptorServletListener.mtl

```diff
@@ -69,14 +71,23 @@ import [javaClassFullNameForAdaptorManager(anAdaptorInterface) /];
 // [protected ('imports')]
 // [/protected]
 
+/**
+ * During the initialization of this ServletListener, the base URI for the OSLC resources produced by this server is configured through the OSLC4J method setPublicURI().
+ * A number of approaches can be used to set this base URI:
```

Comment:
I think this formatting will render badly in Eclipse/Intellij. I think we'd need <p> and ul/li tags here.

Reply:
Can you please suggest what you mean? Not sure I get it

Reply:
```java
/**
 * During the initialization of this ServletListener, the base URI for the OSLC resources produced by this server is configured through the OSLC4J method setPublicURI().
 * <p>
 * A number of approaches can be used to set this base URI:
 * <ol>
 * <li>Set a system environment property of name "LYO_BASE".</li>
 * <li>In the web.xml file, set a {@code context-param} property, with name {@code [javaClassPackageNameForAdaptorServletListener(anAdaptorInterface) /].baseurl}</li>
 * <li>Override the value of the DEFAULT_BASE constant.</li>
 * </ol>
 */
```

Reply:
Thank you. fixe.